### PR TITLE
Adding FractionallySizedBox to the list of useful widgets.

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/3029

Staged: https://sz-flutter-2.firebaseapp.com/docs/development/ui/layout/responsive

(Also removed the "Creating responsive apps" page from the Flutter wiki.)

### CODE REVIEW

File: src/docs/development/ui/layout/responsive.md [hunk 1]
```diff
@@ -48,19 +42,24 @@ apps with responsive design:
...
 
-* [`AspectRatio`][]
-* [`CustomSingleChildLayout`][]
-* [`CustomMultiChildLayout`][]
-* [`FittedBox`][]
-* [`LayoutBuilder`][]
-* [`MediaQuery`][]
-* [`MediaQueryData`][]
-* [`OrientationBuilder`][]
+* [`AspectRatio`][] API and
```
Comment (kwalrath) [inline src/docs/development/ui/layout/responsive.md]: How about something like this, to make the list more natural:

* [`AspectRatio`](...) (also see: [Widget of the Week video](...))
* [`CustomSingleChildLayout`](...)

Comment (kwalrath) [reply src/docs/development/ui/layout/responsive.md]: Oops, the link text for the videos shouldn't all be the same, for accessibility reasons. But you get the gist.

**Comment by kwalrath** [comment]
I guess I also have a question about whether we really need to link to the videos, when the API docs already do so.

**Comment by sfshaza2** [comment]
@kwalrath, please take another look.

Staged: https://sz-flutter-2.firebaseapp.com/docs/development/ui/layout/responsive

**Pull request merged**